### DISCUSSION TITLE
Change the loading indicator (throbber) and change the display of bookmarks in folders

### DISCUSSION THREAD

**Discussion by Someone-G** [black7375/Firefox-UI-Fix#448] [Q&A]
Hello. Tell me how to change the loading indicator (throbber) so that it is in the form of a circle?
Is it possible to change the display of bookmarks (when there are many) in a folder in multiple columns (2, 3, 4...)?
Thank you.

**Comment by black7375** [comment]
Do you have any examples you want? (screenshot)

My understanding is as follows:
-  Loading indicator (throbber)
![chrome throbber](https://user-images.githubusercontent.com/25581533/142760891-4c75317c-cad7-4554-88a1-f700b474c0d0.gif)
- Bookmark folder's childen count like this
![image](https://user-images.githubusercontent.com/25581533/181904351-ac6e0c37-10d6-4069-bc34-26d20d7e1915.png)


In conclusion, I think loading indicator may be possible, but bookmark children counting is close to the impossibility with CSS.

**Reply by Someone-G** [reply]
You have a great loading indicator! I like it! Can this be enabled? How about black? I'm using a light (system) theme. And I want the indicator not to stand out from the general style of Photon.

For old versions of the Firefox browser, people wrote codes to display bookmarks in multiple columns in folders, but now this is not possible? Everyone I asked - they say that in modern realities this cannot be implemented. Will have to get used to...

**Reply by black7375** [reply]
Loading indicator can be added as an option to a `v7.0.0` roadmap.
Multi columns will try, but it won't be easy.

**Reply by Someone-G** [reply]
> Loading indicator can be added as an option to a v7.0.0 roadmap. 

I don't get it... I just installed Firefox-UI-Fix v5.3.0 (Lepton-Photon-Style.zip) and don't know what to do yet.

**Reply by black7375** [reply]
Screenshot is just example.
It is necessary to implement and will be done at `v7.0.0`.

**Comment by Someone-G** [comment]
Hello. I set 7.0.0 and while the loading (reboot) indicator is old. Is it possible to somehow change it to a circle (circulation)?
